pyinter/interval.py

```diff
@@ -195,6 +195,11 @@ def _get_new_lower_upper(self, other, operator):
             else:
                 new_upper = self._upper
         return new_lower, new_upper
+    
+    def empty(self):
```

Comment:
As this is a public method, what do you think about adding a docstring?

Also, what do you think of `is_empty` to make it clear that this returns a bool. `empty` could mean it will "empty the interval"